### DISCUSSION TITLE
An alternative  image download channel in Mainland China

### DISCUSSION THREAD

**Discussion by yaleimeng** [milvus-io/milvus#4736] [Show and tell]
我这边使用国内多个知名docker源下载gpu镜像速度都非常慢，同时下载源码编译也报错，耽误了很多时间。
特将保存之后的tar文件进行分享，便于下载后直接 docker load < milvus_image.tar 节省一些时间和精力。版本为最新0.10.6：
GPU版镜像：https://cloud.189.cn/t/2UN3yuqE3eay
CPU版镜像：https://cloud.189.cn/t/Uze2uqrA7BNn

**Comment by jingkl** [comment]
Our latest version of Milvus is 0.10.6 instead of 1.10.6, thank you for your sharing

**Reply by gujun720** [reply]
I have updated the typo.
